src/issue/screens/issue.screen.js

```diff
@@ -214,14 +220,25 @@ class Issue extends Component {
     />;
 
   render() {
-    const { issue, comments, isPendingComments, navigation } = this.props;
+    const {
+      issue,
+      comments,
+      isPendingComments,
+      isPendingIssue,
+      navigation,
+    } = this.props;
 
     return (
       <ViewContainer>
-        {isPendingComments &&
-          <LoadingContainer animating={isPendingComments} center />}
+        {(isPendingComments || isPendingIssue) &&
+          <LoadingContainer
+            animating={isPendingComments || isPendingIssue}
+            center
+            text={'Retrieving issue'}
```

Comment:
@housseindjirdeh @andrewda I was using the `notifications.screen` as a reference and it contains the text message, but sure thing, the UX should be consistent. I think it's a good idea to put texts in a separate PR. Removed in 5dd58c6 😉